openc3-cosmos-init/plugins/packages/openc3-cosmos-tool-cmdsender/src/tools/CommandSender/CommandSender.vue

```diff
@@ -537,10 +540,10 @@ export default {
     // sent from the history. In that case commandName and paramList are undefined
     // and the api calls handle that.
     sendCmd(targetName, commandName, paramList) {
-      // Store what was actually sent
-      this.targetName = targetName
-      this.commandName = commandName
-      this.paramList = paramList
```

Comment:
This is actually an important bug fix. When I fixed sending hazardous from history I broke sending a command from the button AFTER sending a command from history.